tools/wptrunner/wptrunner/vcs.py

```diff
@@ -54,7 +56,7 @@ def bind_to_repo(vcs_func, repo, log_error=True):
 
 def is_git_root(path, log_error=True):
     try:
-        rv = git("rev-parse", "--show-cdup", repo=path, log_error=log_error)
+        rv = git("rev-parse", "--show-cdup", repo=isomorphic_decode(path), log_error=log_error)
```

Comment:
Could we workaround this inside `git` instead (which is `def inner(...)` in `tools/wptrunner/wptrunner/vcs.py`), where `repo` is used as the value for the `cwd` parameter in popen? That's still a workaround for our types being messed up in this code, but it at least contains the workaround to somewhere where we can start working backwards to fix it.

Reply:
Incidentally, `cwd` is meant to be able to be a byte string, but that was only fixed in 3.8 on Windows >_<.

```
If cwd is not None, the function changes the working directory to cwd before executing the child. cwd can be a string, bytes or path-like object. In particular, the function looks for executable (or for the first item in args) relative to cwd if the executable path is a relative path.

Changed in version 3.6: cwd parameter accepts a path-like object on POSIX.

Changed in version 3.7: cwd parameter accepts a path-like object on Windows.

Changed in version 3.8: cwd parameter accepts a bytes object on Windows.
```

Reply:
Updated. Thank you!